japecheck/analyzer.go

```diff
@@ -179,6 +185,24 @@ func parseRouteDef(kv *ast.KeyValueExpr, pass *analysis.Pass) (*route, bool) {
 					return false
 				}
 				r.response = call.Args[0]
+			case "DecodeForm":
+				t := pass.TypesInfo.Types[call.Args[1]].Type
+				if t == nil {
+					return false
+				} else if p, ok := t.(*types.Pointer); ok {
+					t = p.Elem()
+				}
+				r.params = append(r.params, routeParam{t.String(), call.Args[1]})
```

Comment:
I was thinking `name` would be the `key` argument to `DecodeForm` (i.e. `exprToString(call.Args[0])`) so that we can report e.g. `wrong type for path parameter "host"`

Reply:
(you can simply leave the `name` field empty when parsing the client)